### DISCUSSION TITLE
Finding Turbine for Hydropower Release Schedule

### DISCUSSION THREAD

**Discussion by c-luu** [USACE/cwms-data-api#995] [Q&A] [answered]
How would I find the data behind this website? https://spatialdata.usace.army.mil/hydropower/

Seems to be no BUFORD DAM/LAKE SIDNEY LANIER turbine project (I assume this is the right resource because the endpoint `/cwms-data/projects/{office}/{name}/turbine-changes` has water discharge response data. 

I believe the office info is at:

```
 {
    "stream-location-node": {
      "id": {
        "office-id": "SAM",
        "name": "Buford"
      },
      "stream-node": {
        "stream-id": {
          "office-id": "SAM",
          "name": "Chattahoochee R"
        },
        "station": 348.30000000000007,
        "station-units": "mi"
      }
    },
    "area-units": "mile2",
    "stage-units": "ft"
  },
```

**Comment by MikeNeilson** [comment, marked as answer]
The data api is most likely the right place; however, we're still transitioning to having everything gathered together in one place so it's possible the data it's pushed to our database, or just has a different name.

I'll let the district know about the question so they can help dig in more; unfortunately our search support is also more limited than we'd like.

That said, if you use the time series catalog you can go through the list and see if the public or long name matches.

Also, it appears our main site if having issues, so you can also use https://cwms.sec.usace.army.mil/cwms-data/ and it will have the same data, at least in real-time.

**Reply by c-luu** [reply]
Thank you so much!
